app/src/main/res/layout/individual_exercise_detail.xml

```diff
@@ -10,8 +10,9 @@
     android:orientation="vertical">
```

Comment:
Your build is not running, take a look at the Travis file. It looks like there is an error in your strings values file